overlord/hookstate/hookmgr.go

```diff
@@ -41,6 +41,10 @@ type HookManager struct {
 	state      *state.State
 	runner     *state.TaskRunner
 	repository *repository
+
+	// RunTool is used to run the requested tool. It's exported here for testing
+	// purposes.
+	RunTool func(ToolRequest) (string, string)
```

Comment:
I've removed the test. If there's a better way you'd like me to test this, let me know.